cmd/dashboard/config/config.go

```diff
@@ -71,6 +75,7 @@ type Config struct {
 
 	Notifications           Notifications
 	Chart                   Chart
+	Charts                  string `envconfig:"CHARTS"`
```

Comment:
Make it a struct and implement a parsing interface for it like with `Multiline`.